code/src/pages/Detail.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react'
+import { useParams, useNavigate } from 'react-router-dom'
+import { API_DETAILS } from '../utils/urls'
+import '../css/detail.css'
+
+
+const Detail = () => {
+const [details, setDetails] = useState({});
+const {movieId} = useParams();
+const navigate = useNavigate();
+const onBackButtonClick = () => {
+    navigate(-1);
+}
+
+useEffect (() => {
+    fetch(API_DETAILS(movieId))
+    .then((res) => res.json())
+    .then((data) => setDetails(data))
+    }, [movieId]) 
+
+    return (
+        <section>
+            {details && (
+                <div className="details-container">
+                   <img className="movie-background" src={`https://image.tmdb.org/t/p/w1280${details.backdrop_path}`} alt={details.title} />
```

Comment:
There is something going on with the backdrop in desktop mode, which makes the text disappear and some of the poster img, and it is not possible to scroll down to read. Maybe it could be fixed by trying another img size here instead w1280 or set a max-height on the section?
When we checked in the inspector it seems that the margin for the detail-content and the poster img takes up a lot of the space, so if you decrease the margin and the poster img size things could fit better! 
We struggled with the backdropped to and ended up wrapping all the detail component in the backdrop image as a styled component and sent it in as a props.